UPGRADE-1.14.md

```diff
@@ -662,6 +662,16 @@
 1. The class `Sylius\Bundle\AdminBundle\EventListener\ResourceDeleteSubscriber` has been deprecated and will be removed in Sylius 2.0.
    It will be replaced with the `ResourceDeleteListener`.
 
+1. The following exceptions have been deprecated and will be removed in Sylius 2.0:
+    - `Sylius\Bundle\ApiBundle\Exception\PaymentMethodCannotBeRemoved`
+    - `Sylius\Bundle\ApiBundle\Exception\ProductVariantCannotBeRemoved`
+    - `Sylius\Bundle\ApiBundle\Exception\PromotionCannotBeRemoved`
+    - `Sylius\Bundle\ApiBundle\Exception\PromotionCouponCannotBeRemoved`
+    - `Sylius\Bundle\ApiBundle\Exception\ShippingMethodCannotBeRemoved`
+    - `Sylius\Bundle\ApiBundle\Exception\ProductAttributeCannotBeRemoved`
```

Comment:
It could be ordered by bundles for better readability
```suggestion
    - `Sylius\Bundle\ApiBundle\Exception\PaymentMethodCannotBeRemoved`
    - `Sylius\Bundle\ApiBundle\Exception\ProductAttributeCannotBeRemoved`
    - `Sylius\Bundle\ApiBundle\Exception\ProductVariantCannotBeRemoved`
    - `Sylius\Bundle\ApiBundle\Exception\PromotionCannotBeRemoved`
    - `Sylius\Bundle\ApiBundle\Exception\PromotionCouponCannotBeRemoved`
    - `Sylius\Bundle\ApiBundle\Exception\ShippingMethodCannotBeRemoved`
```

Moreover, I believe it misses the following classes:
    - `Sylius\Bundle\ApiBundle\Exception\ZoneCannotBeRemoved`
    - `Sylius\Bundle\ApiBundle\Exception\ProductCannotBeRemoved`